src/aomusicplayer.cpp

```diff
@@ -32,6 +32,11 @@ void AOMusicPlayer::play(QString p_song, int channel, bool loop,
   else
     newstream = BASS_StreamCreateFile(FALSE, f_path.utf16(), 0, 0, flags);
 
+  if (f_path.startsWith("http"))
+    newstream = BASS_StreamCreateURL(f_path.toStdWString().c_str(), 0, BASS_STREAM_AUTOFREE | BASS_UNICODE | BASS_ASYNCFILE, NULL, NULL);
+  else
+    newstream = BASS_StreamCreateFile(FALSE, f_path.utf16(), 0, 0, flags); 
+
```

Comment:
Ah, gosh. Why did I push a style commit. This is wrong. The stream is being made twice.

Reply:
I tried condensing it by trying an #elseif however it didn't work.